### PR TITLE
User Can View Daily Planner

### PR DESCRIPTION
**Pull request by htjahyon**
image

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+  <head>
+    <title>Week Planner</title>
+    <link rel="stylesheet" href="style.css"/>
+  </head>
+  <body>
+    <h1>Weekly Planner</h1>
+    <section class="week"></section>
+    <section class="add">
+      <p>Add Entry</p>
+    </section>
```
Comment (shadorki) [inline index.html]: a section element should be used to separate your page into sections, it doesnt really make sense to do it here

You want to use a div or a button for your add entry button

File: main.js [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+];
+function returnDays(object) {
+  var dayElement = document.createElement("div");
+  dayElement.className = "day";
+  dayElement.textContent = object.day;
+  return dayElement;
+}
+for(var i=0; i<days.length; i++) {
+  document.querySelector(".week").appendChild(returnDays(days[i]));
+}
```
Comment (shadorki) [inline main.js]: You will not need to dynamically create your week in this application.

File: main.js [hunk 3]
```diff
@@ -0,0 +1,49 @@
...
+  titleElement.style = "background-color: lightgray";
+  tableElement.appendChild(titleElement);
+  for(var i=0; i<8; i++) {
+    var rowElement = document.createElement("tr");
+    var dataElement = document.createElement("td");
+    rowElement.appendChild(dataElement);
+    tableElement.appendChild(rowElement);
+  }
+  return tableElement;
+}
```
Comment (shadorki) [inline main.js]: You only need to create the rows dynamically for its content, the headings will never change.

You will also need thead and tbody elements to be semantically correct.

File: main.js [hunk 4]
```diff
@@ -0,0 +1,49 @@
...
+document.querySelector(".tables").appendChild(returnTable("Time"));
+document.querySelector(".tables").appendChild(returnTable("Description"));
```
Comment (shadorki) [inline main.js]: If you're going to do things in the global scope to start your application you should start it with a `start` function or an `init` function

File: style.css [hunk 5]
```diff
@@ -0,0 +1,47 @@
...
+  display:flex;
+  width:100%;
+  justify-content:center;
+}
+#Time {
+  width:39%;
+  height:80px;
+  border:2px solid;
+}
+#Description {
```
Comment (shadorki) [inline style.css]: No capitalized ids, use camelCase or kebab-case

File: style.css [hunk 6]
```diff
@@ -0,0 +1,47 @@
+h1 {
+  text-align:center;
+}
+.week {
+  display:flex;
+  justify-content:center;
+}
+.day {
+  width:10%;
```
Comment (shadorki) [inline style.css:10]: 10% doesn't work if the parent element does not have a width

**Review by shadorki — CHANGES_REQUESTED** [review]

**Comment by htjahyon** [comment]
image

**Review by shadorki — APPROVED** [review]

**Pull request merged**